### PR TITLE
Fixes #8583 - Renaming id attribute to avoid lookup

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: lib/hammer_cli_katello/content_view_puppet_module.rb [hunk 1]
```diff
@@ -29,11 +29,8 @@ def extend_data(mod)
     class CreateCommand < HammerCLIKatello::CreateCommand
       command_name "add"
 
-      option "--id", "ID", _("id of the puppet module to associate")
-
-      def request_params
-        super.tap { |params| params['uuid'] = params.delete('id') if params['id'] }
-      end
+      option "--id", "ID", _("id of the puppet module to associate"),
+        :attribute_name => "uuid"
```
Comment (komidore64) [inline lib/hammer_cli_katello/content_view_puppet_module.rb]: and this gets put into the request params?

Comment (daviddavis) [inline lib/hammer_cli_katello/content_view_puppet_module.rb]: Yea, it's a parameter in the API.

Comment (komidore64) [inline lib/hammer_cli_katello/content_view_puppet_module.rb]: awesome

**Comment by komidore64** [comment]
ack

**Pull request merged**